### ISSUE TITLE
InivitalValues not displayed with AutoComplete component.

### ISSUE THREAD

**Issue by pzapo** [gerhat/material-ui-formik-components#28]
Passing initialValues to formik Field component with AutoComplete is broken. InitialValues are not rendered on AutoComplete component but when save is clicked then they are available in `values` formik variable.

Here is link to sample code based on your example from documentation:
https://codesandbox.io/s/goofy-wildflower-i3h1r

**Comment by dalborgo** [comment]
The same problem for me.

**Comment by gian788** [comment]
Same issue for me too

**Comment by gerhat** [comment]
Thank you for reporting @pzapo @dalborgo and @gian788.
It is indeed an issue. Will fix asap.

**Comment by shtjefeen** [comment]
@gerhat Did u get this fixed!

**Comment by shtjefeen** [comment]
@gerhat I created a pull request for this one

**Comment by gerhat** [comment]
Issue fixed in v0.6.2.

Here is an example on how to pass initial values to autocomplete.
```javascript
const initialValues = {
  skills: [
    {
      label: 'ASP.NET',
      value: 'ASP.NET',
    },
  ],
}
````
